packages/grid_client/tests/client_loader.ts

```diff
@@ -11,17 +11,18 @@ const mnemonic = env.MNEMONIC;
 const storeSecret = env.STORE_SECRET;
 const ssh_key = fs.readFileSync(os.homedir() + "/.ssh/id_ed25519.pub", "utf-8");
 let config;
+const defaultConfig = JSON.parse(fs.readFileSync(path.join(__dirname, "./config.json"), "utf-8"));
 
 if (!network || !mnemonic || !ssh_key) {
```

Comment:
this won't fix the issue, 
![Screenshot from 2024-07-04 08-10-28](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/387e085f-eec5-4747-b3ec-3c9a6e8783df)

i think its better to revert this changes and mention on docs that all credential must be in one place